hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -502,6 +502,121 @@
         ]
       }
     }
+    { name: "phy_pins_sense",
+      desc: "USB PHY Pin Sense",
```

Comment:
Maybe "USB PHY Pin Sense (for testing)"?

Reply:
I have extended the description to mention the purpose of the registers (debugging and chip testing).